azure-pipelines.yml

```diff
@@ -30,18 +30,18 @@ stages:
       pool:
         vmImage: $(imageName)
       steps:
-      - script: npm install -g pnpm@6.19.0
-        displayName: install pnpm globally
-
-      - script: pnpm install
+      - script: npm ci
         displayName: 'Install dependencies'
 
-      - script: pnpm run build
+      - script: npm run build
         displayName: 'Build'
 
-      - script: pnpm run lint
+      - script: npm run lint
         displayName: 'Lint'
 
+      - script: npm run test:unit
```

Comment:
Add a new step after `test:unit`:
```
npm audit --registry=https://registry.npmjs.org --audit-level=high
```

And check if there are any issues with the audit. If there are issues (vulnerabilities) - try to fix them.

Reply:
added audit step to pipeline